src/screens/People/People.js

```diff
@@ -681,15 +629,35 @@ class PeopleScreen extends React.Component<Props, State> {
     const {
       localContacts,
       chats,
+      theme,
+      goToInvitationFlow,
     } = this.props;
     const inSearchMode = query.length >= MIN_QUERY_LENGTH;
 
     const sortedLocalContacts = sortLocalContacts(localContacts, chats);
     const contact = sortedLocalContacts.find(({ id }) => id === manageContactId);
 
+    const colors = getThemeColors(theme);
+
     return (
       <ContainerWithHeader
-        headerProps={{ noBack: true, leftItems: [{ title: 'People' }] }}
+        headerProps={{
+          noBack: true,
+          leftItems: [{ title: 'People' }],
+          rightItems: [{
+            custom: (
+              <IconButton
+                icon="present"
+                color={colors.positive}
```

Comment:
I don't think it's worth the effort. In case of buttons it makes sense as we have a set of button styles definied in style guide (primary, secondary etc). In case of icon buttons I can't see such styles and probably they won't be reused.